From daeed8363d130d35f4b6c17b3ce3c50565d5fc32 Mon Sep 17 00:00:00 2001
From: Ajay Kumar Gupta <ajay.gupta@ti.com>
Date: Fri, 8 Jun 2012 16:36:05 +0530
Subject: [PATCH 1018/1092] usb: musb: ti81xx: fix wrong timer check during
 suspend

Corrected the wrong check done for musb->otg_workaround instead of
musb->otg_timer for deleting musb->otg_timer. So far no issue is
seen due to this but fixing it anyways.
---
 drivers/usb/musb/ti81xx.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/musb/ti81xx.c b/drivers/usb/musb/ti81xx.c
index 6b0eb9e..d0e7202 100644
--- a/drivers/usb/musb/ti81xx.c
+++ b/drivers/usb/musb/ti81xx.c
@@ -1505,7 +1505,7 @@ static void ti81xx_save_context(struct ti81xx_glue *glue)
 			musb->en_otgw_timer = 1;
 		}
 
-		if (timer_pending(&musb->otg_workaround) &&
+		if (timer_pending(&musb->otg_timer) &&
 					is_otg_enabled(musb)) {
 			del_timer_sync(&musb->otg_timer);
 			musb->en_otg_timer = 1;
-- 
1.7.11.2

